docs/apis-tools/operate-api/authentication.md

```diff
@@ -4,57 +4,141 @@ title: Authentication
 description: "Authentication requirements for accessing the Operate REST API."
 ---
 
-All Operate REST API requests require authentication.
-
-## Authentication for SaaS
-
-### Authentication via JWT access token
```

Comment:
Non-blocking change (NBC): Not sure if we should spell out "JSON Web Token (JWT)", I guess by this point the audience know this and the docs may have already spelled it out?

Reply:
I like this addition. Do you think it is also worth linking to something like https://jwt.io/introduction/ to give more context for people who've never worked with token-based authentication before?

Reply:
oh, this is on a deleted line. I am going to spell them out and link to a resource in the first mention of them instead.